runtime-watcher/tests/e2e/watcher_metrics_test.go

```diff
@@ -0,0 +1,60 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/kyma-project/runtime-watcher/skr/tests/e2e/utils"
+)
+
+var _ = Describe("Watcher Metrics", Ordered, func() {
+	kyma := NewKyma(kymaName, controlPlaneNamespace, kymaChannel,
+		v1beta2.SyncStrategyLocalSecret)
+	GinkgoWriter.Printf("kyma before create %v\n", kyma)
```

Comment:
Is this needed or a leftover from debugging?

Reply:
it was copied from the other test but it is not needed, will remove it